exec/java-exec/src/main/java/org/apache/drill/exec/store/AbstractStoragePlugin.java

```diff
@@ -29,13 +29,20 @@
 import org.apache.drill.exec.planner.PlannerPhase;
 
 import com.google.common.collect.ImmutableSet;
+import org.apache.drill.exec.server.DrillbitContext;
 
 /** Abstract class for StorePlugin implementations.
  * See StoragePlugin for description of the interface intent and its methods.
  */
 public abstract class AbstractStoragePlugin implements StoragePlugin {
 
-  protected AbstractStoragePlugin() { }
+  protected final DrillbitContext context;
+  final String name;
```

Comment:
`protected`. Actually, since you've provided `getFoo()` methods, these can be private.